wgengine/netstack/netstack.go

```diff
@@ -233,7 +232,7 @@ func (ns *Impl) Close() error {
 func (ns *Impl) wrapProtoHandler(h func(stack.TransportEndpointID, stack.PacketBufferPtr) bool) func(stack.TransportEndpointID, stack.PacketBufferPtr) bool {
 	return func(tei stack.TransportEndpointID, pb stack.PacketBufferPtr) bool {
 		addr := tei.LocalAddress
-		ip, ok := netip.AddrFromSlice(net.IP(addr))
+		ip, ok := netip.AddrFromSlice(net.IP(addr.AsSlice()))
```

Comment:
The `net.IP()` conversion is no longer needed here

Reply:
Resolved [here](https://github.com/tailscale/tailscale/pull/9252/commits/a7f8c6848096917d266aa24f2f73b9f506fcec26). Thanks.